src/main/GreaterThanConstantDiscountedPurchase.java

```diff
@@ -0,0 +1,26 @@
+package main;
+
+public class GreaterThanConstantDiscountedPurchase extends AbstractPurchase {
+    private final double discount;
+    private final int exceedQuantity = 15;
```

Comment:
```suggestion
    private final static int EXCEED_QUANTITY = 15;
```
this is **class** constant.